oaknational/Oak-Web-Application
# chore: new globe image

## Description

- adds new globe image


## How to test
- go to https://deploy-preview-1859--oak-web-application.netlify.thenational.academy/beta/teachers-2023/key-stages/ks1/subjects
- should have a new globe image (without extra island)
